pages/gallery.tsx

```diff
@@ -1,3 +1,142 @@
+import { Dialog, ImageList, ImageListItem } from '@mui/material'
+
+import { useEffect, useState } from 'react'
+
 export default function GalleryPage() {
-    return <div>Gallery Page holder</div>
+    const [imageUrlArray, setImageUrlArray] = useState([])
+    const [imageSrc, setImgSrc] = useState('')
+    const [open, setOpen] = useState(false)
+    const [nextImageSetUrl, setNextImageSetUrl] = useState('')
+    const getThis = async () => {
```

Comment:
This should be moved to the lib/db functions for the gallery page, and called on load of the application in the getCms data function. No need to query this only when the user lands on this page. also, move all of the state to redux